bundles/lpar/jammy-zed-edge.yaml

```diff
@@ -0,0 +1,465 @@
+      network-manager: Neutron
+      worker-multiplier: 0.25
+    to:
+    - lxd:2
+    channel: *openstack-charm-channel
+  nova-compute:
+    annotations:
+      gui-x: '250'
+      gui-y: '250'
+    charm: ch:nova-compute
+    num_units: 3
+    options:
+      config-flags: default_ephemeral_format=ext4
+      openstack-origin: *openstack-origin
+      enable-live-migration: true
+      enable-resize: true
+      migration-auth-type: ssh
+    to:
+    - '1'
+    - '2'
+    - '3'
+    channel: *openstack-charm-channel
+  dashboard-mysql-router:
+    charm: ch:mysql-router
+    options:
+      source: *openstack-origin
+    channel: *mysql-charm-channel
+  openstack-dashboard:
+    annotations:
+      gui-x: '500'
+      gui-y: '-250'
+    charm: ch:openstack-dashboard
+    num_units: 1
+    options:
+      openstack-origin: *openstack-origin
+    to:
+    - lxd:0
+    channel: *openstack-charm-channel
+  rabbitmq-server:
+    annotations:
+      gui-x: '500'
+      gui-y: '250'
+    charm: ch:rabbitmq-server
+    num_units: 1
+    to:
+    - lxd:0
+    channel: 3.9/edge
+  vault-mysql-router:
+    charm: ch:mysql-router
+    options:
+      source: *openstack-origin
+    channel: *mysql-charm-channel
+  vault:
+    charm: ch:vault
+    num_units: 1
+    to:
+    - 'lxd:4'
+    channel: 1.7/edge
+  swift-proxy:
+    charm: ch:swift-proxy
+    num_units: 1
+    options:
+      zone-assignment: manual
+      replicas: 5
+      swift-hash: 62ce298d-cd4a-4087-b7f3-0c71df0127e0
+      openstack-origin: *openstack-origin
+    to:
+      - "lxd:0"
+    channel: *openstack-charm-channel
+  swift-storage-z1:
+    charm: ch:swift-storage
+    num_units: 1
+    options:
+      zone: 1
+      block-device: /mnt/swift/swift_dist.img|20G
+      overwrite: "false"
+      openstack-origin: *openstack-origin
+    to:
+    - '0'
+    channel: *openstack-charm-channel
+  swift-storage-z2:
+    charm: ch:swift-storage
+    num_units: 1
+    options:
+      zone: 2
+      block-device: /mnt/swift/swift_dist.img|20G
+      overwrite: "false"
+      openstack-origin: *openstack-origin
+    to:
+    - '1'
+    channel: *openstack-charm-channel
+  swift-storage-z3:
+    charm: ch:swift-storage
+    num_units: 1
+    options:
+      zone: 3
+      block-device: /mnt/swift/swift_dist.img|20G
+      overwrite: "false"
+      openstack-origin: *openstack-origin
+    to:
+    - '2'
+    channel: *openstack-charm-channel
+  swift-storage-z4:
+    charm: ch:swift-storage
+    num_units: 1
+    options:
+      zone: 4
+      block-device: /mnt/swift/swift_dist.img|20G
+      overwrite: "false"
+      openstack-origin: *openstack-origin
+    to:
+    - '3'
+    channel: *openstack-charm-channel
+  swift-storage-z5:
+    charm: ch:swift-storage
+    num_units: 1
+    options:
+      zone: 5
+      block-device: /mnt/swift/swift_dist.img|20G
+      overwrite: "false"
+      openstack-origin: *openstack-origin
+    to:
+    - '4'
+    channel: *openstack-charm-channel
```

Comment:
It would be nice, but obviously not required, if the non-ovn and ovn bundles were easier to diff against each other and just see the ovn/neutron differences.

Reply:
yeah, we may want to make this set of bundles deployable by zaza (e.g. `tox -e deploy -- jammy-zed-ovn`) so we can have an overlay.